### DISCUSSION TITLE
内容处于加密状态

### DISCUSSION THREAD

**Discussion by Sleep312** [pjy612/SteamManifestCache#20] [Question / Answer] [answered]
最近的版本开始 我发现很多游戏都是能成功解锁 但是下载不来 我用其他入库工具就正常 是清单库的原因？但以前的版本当时可以正常下载 很迷 希望作者大大能修复

**Comment by Sleep312** [comment]
例 地平线4

**Reply by ZEERDEER** [reply, marked as answer]
请检查自身问题，SAI版本 2.13
![QQ图片20231202111408](https://github.com/pjy612/SteamManifestCache/assets/10573857/fa16e4f5-9a1d-4c78-91f7-5e2cce5a750d)

**Comment by pjy612** [comment]
怎么说呢。。。清单更新不及时是很正常的事情。。。
有的用能玩就行了。。。
如果 其他入库工具 能用，你就用其他工具。
本质都一样的 最后能玩就行。
另外也说了推荐 入一个玩一个。。。“不存在的东西”搞多了 并发症 更多。
如果以前玩过但是 突然更新 都没清单 可以把 SAI_xxxx_D.st 删了。一般能直接跳过后续更新 让当下能玩游戏。
最后 忠告 别 TMD 搞联机
